### DISCUSSION TITLE
please put ethereum on exchange

### DISCUSSION THREAD

**Discussion by badbabil** [revoxhere/duino-coin#432] [Ideas]
There is no way to transfer the coin in to the indian currency or in usd too . it will be great if you  add ethereum exchange

**Comment by BastelPichi** [comment]
> There is no way to transfer the coin in to the indian currency or in usd too . it will be great if you add ethereum exchange

You can exchange to usd on node -s exchange, trade it on the discord server or exchange to bitcoin cash and than to paypal/etherium on duco exchange...

**Comment by revoxhere** [comment]
Ethereum, like Bitcoin, has enormous fees. That won't be happening any time soon

**Reply by badbabil** [reply]
okay buddy , still supporting your work dude.
